src/dotnet/commands/dotnet-build/Program.cs

```diff
@@ -5,21 +5,33 @@
 using Microsoft.DotNet.Cli.CommandLine;
 using Microsoft.DotNet.Cli.Utils;
 using Microsoft.DotNet.Tools.MSBuild;
+using System.Diagnostics;
+using System;
+using Microsoft.DotNet.Cli;
 
 namespace Microsoft.DotNet.Tools.Build
 {
     public class BuildCommand
     {
-        public static int Run(string[] args)
+        private MSBuildForwardingApp _forwardingApp;
+
+        private BuildCommand() { }
```

Comment:
This constructor is not needed. Since you have another constructor with parameters, there's no parameterless public constructor assumed.